atom/browser/api/atom_api_extension.cc

```diff
@@ -220,6 +222,33 @@ void Extension::Enable(const std::string& extension_id) {
 }
 
 // static
+void Extension::Remove(const std::string& extension_id) {
+  const extensions::Extension* extension =
+    GetInstance()->extensions_.GetByID(extension_id);
+  if (extension) {
```

Comment:
yes, but there are other checks that should happen and I think it's better to keep all of that within `AtomExtensionSystem`. See `ExtensionService::UninstallExtension` and `ExtensionService::UnloadExtension`. We don't need everything in there, but I think we should keep the basic structure the same because it's much easier to follow changes in libchromiumcontent that way